libs/openchallenges/challenge/src/lib/challenge-overview/challenge-overview.component.html

```diff
@@ -76,7 +76,13 @@ <h3 class="section-title">Hosted By</h3>
     </div>
   </div> -->
   <hr />
-  <span class="mat-caption created-updated-dates"
-    >Added to the OC on {{ challenge.createdAt }} and last modified on {{ challenge.updatedAt }}
+  <span class="mat-caption created-updated-dates">
+    Added to the OC on {{ challenge.createdAt }} and last modified on {{ challenge.updatedAt }} //
+    See something missing or not up-to-date?
+    <a
+      href="https://docs.google.com/forms/d/e/1FAIpQLSdkgAVOuof-tE4PvCnpPbbGnQvvMhLuTkXikDXTbsCHqcMVSw/viewform?usp=pp_url&entry.1040159972={{challenge.slug}}"
```

Comment:
The challenge slug is technically mutable. This could lead to a problem where two edits are submitted for the same challenge - from potentially different persons - and the first edit asks to replace the slug.

The challenge ID immutable? If yes, it would be safer to use it.

Reply:
We could still include the challenge slug or title - prepopulated - to provide submitter with a human-readable way to remember what challenge is being edited.

Reply:
If only Google Forms allowed for read-only fields 😢 

> Is the challenge ID immutable? If yes, it would be safer to use it.

Good idea.